### PR TITLE
Update our default blog post image

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

I've removed the script that automatically generated the tile images when none were set and replaced it with an image that better aligns with our new colour scheme and style.

## Related Issue(s)

closes [#1793](https://github.com/FlowFuse/website/issues/1793)

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Yndira-E** [comment]
Hi @knolleary,
I've commented out the images in some files—the ones that used to require the auto-generated image. This was done to check if it worked and to see how they looked in context. I thought it could be useful to maintain them this way for the review. Could you please take a look as is? I'll uncomment those images afterward.

File: src/blog/2024/03/flowfuse-self-hosted-starter-resource-limits.md
Comment (ZJvandeWeg) [inline src/blog/2024/03/flowfuse-self-hosted-starter-resource-limits.md:1]: This is the example I take it?

**Comment by Yndira-E** [comment]
@ZJvandeWeg yes, that's one of them. Here's a list:
https://deploy-preview-1798--flowforge-website.netlify.app/blog/
https://deploy-preview-1798--flowforge-website.netlify.app/blog/2024/03/flowfuse-self-hosted-starter-resource-limits/
https://deploy-preview-1798--flowforge-website.netlify.app/webinars/ (different default images for the next and past webinars)
https://deploy-preview-1798--flowforge-website.netlify.app/customer-stories/

File: src/community/newsletter.njk [hunk 1]
```diff
@@ -32,15 +32,16 @@ hubspot:
                 </div>
                 <div class="flex flex-col md:flex-row">
                     <div class="pr-2 md:w-1/3">
-                        {% if item.data.image %}
-                            <div class="w-full h-auto mb-10">
+                        <div class="w-full h-auto mb-10">
```
Comment (ZJvandeWeg) [inline src/community/newsletter.njk:35]: Can we make this a partial or shortcode? Not sure what's more appropriate on a tehcnical level, but feels like if we DRY our code, it's easier going forward

Ref: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Comment by knolleary** [comment]
Looks good - but will echo ZJ's comment on whether we can now pull that common piece that inserts the right image into a single place that can be reused.

**Comment by Yndira-E** [comment]
I have created a shortcode that works for most of the templates. However, the templates involving customer-stories are not exactly the same across all files. The exceptions are /node-red and /landing/cloud-signup/. The latter is temporary, so I decided not to create a partial for it, as it will eventually be unnecessary. However, I'm open to feedback if we think it's needed.

I've left the image paths commented out for now so you can verify that everything is working correctly. Please let me know if everything looks good on your end, and I'll proceed to uncomment the image paths.

**Review by knolleary — APPROVED** [review]
Looks good to me

**Review by ZJvandeWeg — APPROVED** [review]

File: src/blog/2024/03/looking-towards-node-red-4.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ subtitle: A look at what is coming in Node-RED 4.0
 description: A look at what is coming in Node-RED 4.0
 date: 2024-03-07
 authors: ["nick-oleary"]
-image: /blog/2024/03/images/looking-towards-node-red-4.png
+image: #/blog/2024/03/images/looking-towards-node-red-4.png
```
Comment (Yndira-E) [inline src/blog/2024/03/looking-towards-node-red-4.md:7]:
```suggestion
image: /blog/2024/03/images/looking-towards-node-red-4.png
```

File: src/blog/2024/03/flowfuse-self-hosted-starter-resource-limits.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ subtitle: The first 5 Node-RED runtimes are part of the starter package going fo
 description: The first 5 Node-RED runtimes are part of the starter package going forward
 date: 2024-03-14
 authors: ["zeger-jan-van-de-weg"]
-image: /blog/2024/03/images/starter-tier-resource-limits.png
+image: #/blog/2024/03/images/starter-tier-resource-limits.png
```
Comment (Yndira-E) [inline src/blog/2024/03/flowfuse-self-hosted-starter-resource-limits.md:7]:
```suggestion
image: /blog/2024/03/images/starter-tier-resource-limits.png
```

File: src/customer-stories/un-wmo-nr-data-sharing.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: World Meteorological Organization (WMO) Uses Node-RED to Modernize Sharing of Earth-System Data
-image: /images/stories/un-wmo.jpg
+image: #/images/stories/un-wmo.jpg
```
Comment (Yndira-E) [inline src/customer-stories/un-wmo-nr-data-sharing.md:3]:
```suggestion
image: /images/stories/un-wmo.jpg
```

File: src/webinars/2024/balena.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Enterprise-grade Edge Device Fleet Management with balena and FlowFuse"
 subtitle: Join FlowFuse's Grey Dziuba and balena's Marc Pous to learn about Fleet Management.
-image: /images/webinars/enterprise-grade-edge-device-fleet-management-balena-and-flowfuse-webinar-january-2024.jpg
+image: #/images/webinars/enterprise-grade-edge-device-fleet-management-balena-and-flowfuse-webinar-january-2024.jpg
```
Comment (Yndira-E) [inline src/webinars/2024/balena.md:4]:
```suggestion
image: /images/webinars/enterprise-grade-edge-device-fleet-management-balena-and-flowfuse-webinar-january-2024.jpg
```

File: src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Building Unified Namespace using Node-RED and MQTT
 subtitle: Join Grey Dziuba, OT Data and Community Strategist, and Kudzai Manditereza, Developer Advocate at HiveMQ, as they discuss MQTT and Node-RED and how it pertains to the Unified Namespace, UNS.
-image: /images/webinars/building-unified-namespace-uns-using-node-red-and-mqtt-webinar-2024-march.jpg
+image: #/images/webinars/building-unified-namespace-uns-using-node-red-and-mqtt-webinar-2024-march.jpg
```
Comment (Yndira-E) [inline src/webinars/2024/building-unified-namespace-using-nodered-mqtt.md:4]:
```suggestion
image: /images/webinars/building-unified-namespace-uns-using-node-red-and-mqtt-webinar-2024-march.jpg
```

File: src/customer-stories/leveraging-node-red-and-flowfuse-to-revolutionize-irrigation.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Leveraging Node-RED and FlowFuse to Revolutionize Irrigation
-image: /images/stories/pidd-view.png
+image: #/images/stories/pidd-view.png
```
Comment (Yndira-E) [inline src/customer-stories/leveraging-node-red-and-flowfuse-to-revolutionize-irrigation.md:3]:
```suggestion
image: /images/stories/pidd-view.png
```

**Pull request merged**